src/ansys/motorcad/core/geometry_fitting.py

```diff
@@ -0,0 +1,200 @@
+"""Unit containing functions for drawing geometry from a list of points."""
+
+from copy import copy
+from enum import Enum
+
+from ansys.motorcad.core.geometry import Arc, Coordinate, EntityList, Line
+
+
+class _EntityType(Enum):
+    line = 0
+    arc = 1
+
+
+class _TestEntity:
```

Comment:
Not sure I partically like this class name, but unsure of what it could be updated to

Reply:
Me neither - open to suggestions :)